commands.json

```diff
@@ -6830,6 +6830,10 @@
         "group": "connection",
         "complexity": "O(1)",
         "history": [
+            [
+                "8.1.0",
+                "A new `availability_zone` field has been added into the response if the `availability_zone` config is set."
```

Comment:
This file `commands.json` was probably generated from valkey (or redis) using one of the scripts under utils/ in the valkey repo. I don't know exactly how it is done. (Probably we should delete it, because I think it is not up to date.) We don't use this file for anything in the website or man pages. We use the information directly from Valkey instead.

What to update instead is the HELLO page: https://github.com/valkey-io/valkey-doc/blob/main/commands/hello.md

Reply:
Oh, I see. But I think the content of the hello.md is fine and I'd like to update the History part instead:
<img width="907" alt="image" src="https://github.com/user-attachments/assets/c70d11a2-929b-4ef9-8b17-81720dd9281a" />

So I guess what I should do is go back to the main repo and update the https://github.com/valkey-io/valkey/blob/3b52186b6a6199884edf447c52828d5ca882142f/src/commands/hello.json

Reply:
The updated PR is https://github.com/valkey-io/valkey/pull/1524.